js/about/preferences.js

```diff
@@ -1071,6 +1174,7 @@ class AboutPreferences extends React.Component {
     if (overlayName === 'addFunds' && isVisible === false) {
       // Hide the child overlay when the parent is closed
       stateDiff['bitcoinOverlayVisible'] = false
+      stateDiff['paymentHistoryOverlayVisible'] = false
```

Comment:
i don't think this line is necessary. the payment history overlay is unrelated to the add funds overlay.